src/openrct2-ui/windows/EditorObjectiveOptions.cpp

```diff
@@ -458,24 +458,24 @@ static void window_editor_objective_options_arg_1_increase(rct_window* w)
         case OBJECTIVE_PARK_VALUE_BY:
         case OBJECTIVE_MONTHLY_RIDE_INCOME:
         case OBJECTIVE_REPAY_LOAN_AND_PARK_VALUE:
-            if (gScenarioObjective.Currency >= MONEY(2000000, 00))
+            if (gScenarioObjective.Currency >= 2000000.00_GBP)
             {
                 context_show_error(STR_CANT_INCREASE_FURTHER, STR_NONE, {});
             }
             else
             {
-                gScenarioObjective.Currency += MONEY(1000, 0);
+                gScenarioObjective.Currency += 1000.0_GBP;
```

Comment:
```suggestion
                gScenarioObjective.Currency += 1000.00_GBP;
```

Please check the whole diff and make sure there are no more occurrences of `.0_GBP`.